.gradient_ignore.exs

```diff
@@ -0,0 +1,32 @@
+[
+  ## still to be fixed
+
+  # ?
+  {"lib/gradient/elixir_type.ex:70", {:type_error, :mismatch}},
+
+  "lib/gradient/elixir_type.ex:38",
+
+#  "lib/gradient/debug.ex:58", ****
```

Comment:
I see that this is properly fixed, so we don't have to keep it in the ignore file anymore.